sonar-db/src/main/java/org/sonar/db/version/MigrationStepModule.java

```diff
@@ -171,6 +172,7 @@ protected void configureModule() {
       UpdateUsersExternalIdentityWhenEmpty.class,
 
       // 5.6.4
+      FixProjectUuidOfDeveloperProjects.class,
```

Comment:
merging into 6.2 or master, you should make sure these two classes do not appear twice in the module